iqmotion/iq_devices/iq_module.py

```diff
@@ -20,23 +20,33 @@ class IqModule:
     _DEFAULT_VELOCITY_CLIENT_ENTRY = ""
     _DEFAULT_VOLTS_CLIENT_ENTRY = ""
 
-    def __init__(self, com: Communicator, module_idn=0, clients_path=None):
+    def __init__(self, com: Communicator, 
+                module_idn=0,  
+                clients_path: str=None, 
+                extra_clients: list=None, 
+                module_file_path: str=None):
         
         self._client_dict = {}
         self._com = com
         self._module_idn = module_idn
 
         # Load the Module JSON (Base, Vertiq, Fortiq, etc...)
-        self._module_file_dict = IqModuleJsonParser.parse_default_modules(
-            self._MODULE_FILE_NAME
-        )
-        
-        # Deprecated: Add a custom module 
-        # self._module_file_dict = IqModuleJsonParser.parse_module(module_file_path)
+        if module_file_path is None:
+            self._module_file_dict = IqModuleJsonParser.parse_default_modules(
+                self._MODULE_FILE_NAME
+            )
+        # this only occurs from custom IQ Module
+        else:
+            self._module_file_dict = IqModuleJsonParser.parse_module(module_file_path)
 
         # Load the default clients from the Module JSON
         self._create_clients(self._module_file_dict)
 
+        # Load a list of json paths
+        if extra_clients is not None:
```

Comment:
Was this also part of custom module, or is it in here for some other reason?

Reply:
This allows for users to create an IqModule and pass in additional clients, such as clients that they have locally and not part of the API. I believe some people do use this unrelated to custom_iq_module.